# Jetstream proposal
 
Jetstream access for the Bioinformatics Training Workshop at the Society of Environmental Toxicology and Chemistry - North America (SETAC) meeting

**Abstract**
 
Transcriptomic analysis is valuable for understanding underlying molecular mechanisms of responses to environmental stressors, particularly in Environmental Toxicology. However, computational analysis of mRNA sequencing data continues to be a barrier for discoveries. Recently, transcriptomic analysis has become a major interest in the field, but few young scientists are equipped with basic computational skills necessary to accomplish such analyses.

The 39th Annual Meeting of the Society of Environmental Toxicology and Chemistry - North America will be held in Sacramento, CA this year from 4-8 November, 2018 with an accompanying skills workshop, “mRNAseq Data Workshop: De Novo Assembly and Differential Expression Analysis”. This hands-on workshop will offer participants the opportunity to overcome this initial computational analysis barrier. With this course, we aim to provide participants with the pipeline framework and ability to use the software tools for a differential expression analysis of transcriptomic data. We will cover experimental design considerations, transcriptome assembly and differential expression data analysis with a sample dataset. Many of the researchers attending this workshop are early career scientists, who have relevant questions they will continue exploring within the scope of the National Science Foundation.

This proposal is requesting access to Jetstream resources, which will provide computational capacity for the 50 participants in this bioinformatics training workshop. Access to the Jetstream platform will allow all participants to have uniform computational environments with a workshop-specific software installation image, which will reduce the technical difficulties associated with computing in a classroom setting. Jetstream-specific bioinformatics tutorials and software installation image will be adapted from those that were developed and tested by the instructors for this workshop, who were part of the Data Intensive Biology Summer Institute held at UC Davis in 2017 and 2018 led by Titus Brown.
 
* Elias Oziolor, emoziolor@ucdavis.edu
* Lisa Johnson, ljcohen@ucdavis.edu 
* Jane Park, jajpark@ucdavis.edu

**Resource Justification**

We are requesting 50 m.medium instances with 6 cores, 16 GB RAM, and 130 GB VM space each for each instructor and student for 5 days. The total request is for 36,000 service units (6 cores * 24 hrs/day * 5 days * 50 people). Persistent storage beyond the duration is not necessary for this training workshop.
These calculations are based on a similar but much longer running workshop run by C. Titus Brown with the similar number of attendees.
Resources: IU/TACC (Jetstream)
Keywords: Sequencing, Bioinformatics, Training
Principal investigator: E. Oziolor

**Field of science** 

Genomics
 


